packages/matrix/tests/create-realm.spec.ts

```diff
@@ -1,53 +1,21 @@
-import { test, expect } from '@playwright/test';
-import {
-  synapseStart,
-  synapseStop,
-  type SynapseInstance,
-  registerUser,
-} from '../docker/synapse';
-import {
-  startServer as startRealmServer,
-  type IsolatedRealmServer,
-  appURL,
-} from '../helpers/isolated-realm-server';
+import { test, expect } from './fixtures';
+import { appURL } from '../helpers/isolated-realm-server';
 import {
   clearLocalStorage,
   createRealm,
-  login,
-  registerRealmUsers,
-  setupUserSubscribed,
+  createSubscribedUserAndLogin,
 } from '../helpers';
 
 test.describe('Create Realm via Dashboard', () => {
-  let synapse: SynapseInstance;
-  let realmServer: IsolatedRealmServer;
-
-  test.beforeEach(async () => {
-    // synapse defaults to 30s for beforeEach to finish, we need a bit more time
-    // to safely start the realm
-    test.setTimeout(120_000);
-    synapse = await synapseStart({
-      template: 'test',
-    });
-    await registerRealmUsers(synapse);
-    realmServer = await startRealmServer();
-    await registerUser(synapse, 'user1', 'pass');
-  });
-
-  test.afterEach(async () => {
-    await realmServer?.stop();
-    await synapseStop(synapse.synapseId);
-  });
```

Comment:
I appreciate that so much of this boilerplate is extracted/superfluous now, its accumulation has been bothering me